### DISCUSSION TITLE
Wrong gps position affecting "visited" dashboard

### DISCUSSION THREAD

**Discussion by djpalmis** [teslamate-org/teslamate#2699] [Q&A]
I was playing around with the dashboards, and I noticed that a wrong route was logged.
Is there a way to find out in which day this was logged?
Is there a way to delete the wrong gps position?

I think that "fake trip" alsoaffectst my statistics, right?

![Screenshot_20220612-100354_Samsung Internet](https://user-images.githubusercontent.com/402122/173223566-5394f33d-9e97-438c-9092-86157af8163c.jpg)

**Comment by DrMichael** [comment]
You could remove it from this view by finding and deleting it in the _`positions`_ table.

I don't know though, how it affects the `tables` drives and `charges`, which both use position ids. Perhaps just make a backup, delete it and look, what happens.

**Reply by djpalmis** [reply]
I'll try to follow your tips!
Thank You

**Comment by cwanja** [comment]
Will follow up on this, as I had this situation today. Let me start off by saying  **always backup your data!**

Easiest way to do this is to drop into your database however you installed it. From there, run the following command:
`select id, date, latitude, longitude from positions where drive_id = '###' order by id asc;` where ### is the ID from the drive ID in the URL. 

Then identify the `positions` row IDs that are causing the out of band data - either based upon the timestamps against the drive details dashboard or by plugging the longitude / latitude into your preferred map provider. Then run the following delete command:
`delete from positions where CAST(id as integer) between 1234 and 9876;` where `1234` is the **starting** point of positions records you want deleted and `9876` is the **ending** point.

No issues to the main Drives dashboard or Charges dashboard. Visited dashboard removes the erroneous lines. Seems all fine, at the moment. Will monitor it.

Again, **always backup your data!**
